lib/vanagon/project.rb

```diff
@@ -101,6 +102,17 @@ def get_replaces
       replaces.flatten.uniq
     end
 
+    # Collects all of the conflicts for the project and its components
+    def get_conflicts
+      conflicts = []
+      # Including replaces in the conflicts list helps to ensure
+      # compatibility
+      conflicts.push get_replaces
+      conflicts.push @conflicts.flatten
+      conflicts.push @components.map(&:conflicts).flatten
+      conflicts.flatten.uniq
```

Comment:
(PPS. I'm stir-crazy, so I'm going to check out your branch and see if my suggestion passes the tests you wrote. tests++!)